dist/udb3-angular.css

```diff
@@ -3259,6 +3259,9 @@ form.booking-period table {
 .extra-beschrijving .description-info {
   font-size: small;
 }
+.extra-beschrijving .description-tips p {
```

Comment:
Maybe we should change this class selector to something in english?

Reply:
Agreed that it would be a nice to have, but on the other hand that class already existed before so changing it might have other side effects. And I'm not sure if it's worth going the extra mile for minor things like this given that the Angular app has a limited lifetime ahead of it. So personnally I wouldn't block this because of that.